### PR TITLE
[stable/docker-registry] Add affinity for pod assignment

### PR DESCRIPTION
**Pull request by md-mq**
#### What this PR does / why we need it:

This PR adds affinity for for pod assignment for [stable/docker-registry]

#### Special notes for your reviewer:

No breaking changes, this is PR just exposes std kubernetes pod/node scheduling mechanism. The chart already has toleration and node selectors, and this PR extends the pod assignment with affinity.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/docker-registry/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Docker Registry
 name: docker-registry
-version: 1.7.1
+version: 1.7.2
```
Comment (davidkarlsen) [inline stable/docker-registry/Chart.yaml]:
```suggestion
version: 1.8.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by mouradmourafiq** [comment]
@redbaron  @jpds any comments on this?

**Pull request closed without merging**